pom.xml

```diff
@@ -87,6 +87,19 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>cobertura-maven-plugin</artifactId>
```

Comment:
JaCoCo seems to offer more functionality. It also appears to be the more active project.
I will make a switch to JaCoCo.